sonic_platform_base/sfp_base.py

```diff
@@ -304,3 +304,39 @@ def set_power_override(self, power_override, power_set):
             False if not
         """
         raise NotImplementedError
+
+    def read_eeprom(self, offset, num_bytes):
+        """
+        read eeprom specfic bytes for a given port_num and device id
+        beginning from a random offset with size as num_bytes
+
+        Args:
+             offset :
+                     Integer, the offset from which the read transaction will start
+             num_bytes:
+                     Integer, the number of bytes to be read
+
+        Returns:
+            bytes, raw sequence of bytes which are read from the offset of size num_bytes
+        """
+        raise NotImplementedError
+
+    def write_eeprom(self, offset, num_bytes, write_buffer):
+        """
+        write eeprom specfic bytes for a given port_num and device id
```

Comment:
port_num and device id are no longer given here

Reply:
fixed